test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -146,28 +146,24 @@ json_spirit::mValue doTransitionTest(json_spirit::mValue const& _input, bool _fi
 
 json_spirit::mValue doBlockchainTestNoLog(json_spirit::mValue const& _input, bool _fillin)
 {
-	json_spirit::mValue v = _input; // TODO: avoid copying and add only valid fields to the new object.
-	map<string, json_spirit::mObject> tests;
-	vector<decltype(v.get_obj().begin())> erase_list;
+	json_spirit::mValue v = json_spirit::mObject();
```

Comment:
No, I'm converting an `mObject` to `mValue`.  This function returns an `mValue`.